beprof/curve.py

```diff
@@ -113,7 +113,8 @@ def change_domain(self, domain):
             raise ValueError('in change_domain():' 'the old domain does not include the new one')
 
         y = np.interp(domain, self.x, self.y)
-        obj = self.__class__(np.stack((domain, y), axis=1), **self.__dict__['metadata'])
+        # np.dstack(...)[0] is used to extract nested array (previously used np.stack which behaved different)
+        obj = self.__class__(np.dstack((domain, y))[0], **self.__dict__['metadata'])
```

Comment:
Like "stack is available from numpy 1.10 and to preserve compatibility we use older function which does the same thing as np.stack((arrays), axis=1)" ?